Source/ReturnsExtensions.cs

```diff
@@ -24,6 +24,18 @@ public static class ReturnsExtensions
 		}
 
 		/// <summary>
+		/// Specifies the value to return from an asynchronous method.
+		/// </summary>
+		/// <typeparam name="TMock">Mocked type.</typeparam>
+		/// <typeparam name="TResult">Type of the return value.</typeparam>
+		/// <param name="mock">Returns verb which represents the mocked type and the task of return type</param>
```

Comment:
I realise that you copy-pasted these documentation comments from the existing `Task<T>` methods, so this is more of a comment than asking for a change:

1. This parameter's name should have been `setup`, not `mock`: The `.Returns` "verb" acts on a setup of some mock, not on the mock itself. Unfortunately, renaming this parameter everywhere is theoretically a breaking change (albeit one unlikely to affect anyone, since we're talking of the `this` parameter). Let's leave it as is, better to be consistent everywhere.

2. I wonder if anyone readily understands that complicated summary description. :-) I would personally prefer something simpler, e.g. "The setup whose return value is configured." Feel free to change the description to something sensible *everywhere*, if you're in the mood & can spare the time.

Reply:
On item 2., you're still referring to `<param name="mock">` right? I read the param description several times and I think I finally get it haha. But since it is an extension method and most of the time extension methods are used with extension method syntax instead of as static method calls where you pass in the `ISetup` so most people using these methods won't actually see the comment in the IDE. All the same though, your suggestion slightly tweaked "The setup whose return value is _to be_ configured." seems to make sense to me. I am just a little anxious about not finding _every_ instance as you asked (assuming these kind of comments exist in places other than just the Task/ValueTask extensions).

Reply:
With *everywhere* I actually meant only "everywhere in that particular class". Sorry if that wasn't clear enough. Since it's really a detail that doesn't make a huge difference, like you correctly say, I went ahead &  merged your PR without this. :)